sklearn/utils/tests/test_extmath.py

```diff
@@ -330,6 +331,31 @@ def test_randomized_svd_sign_flip():
         assert_almost_equal(np.dot(v2.T, v2), np.eye(2))
 
 
+def test_randomized_svd_sign_flip_with_transpose():
+    # Check if flipping is always done based on u
+    def max_loading_is_positive(u, v):
+        # u_based, v_based
+        return((np.abs(u).max(axis=0) == u.max(axis=0)).all(),
```

Comment:
Nitpick: a space after a return seems more usual.

Maybe add a comment explaining what is going on in this computation. It is not so easy to grok even if I may have written it originally, so imagine what it will be like in a few months.

My personal attempt: the values maximising np.abs over the rows (resp. columns) should all be positive for u (resp. v). Not that great so try to improve it !